datafusion/core/tests/sql/group_by.rs

```diff
@@ -496,53 +496,49 @@ async fn group_by_date_trunc() -> Result<()> {
 
 #[tokio::test]
 async fn group_by_largeutf8() {
-    {
-        let ctx = SessionContext::new();
+    let ctx = SessionContext::new();
```

Comment:
This change just removes an extra level of (unecessary) indent

It is easier to see the change with https://github.com/apache/arrow-datafusion/pull/2333/files?w=1